metadata-ingestion/docs/sources/mysql/mysql.md

```diff
@@ -0,0 +1,7 @@
+### Prerequisites
```

Comment:
Are there supposed to be other contents?

Reply:
Other content is auto-generated.